cases/world_cereal/presto_feature_extractor.py

```diff
@@ -0,0 +1,19 @@
+import xarray as xr
+
+from openeo_gfmap.features import PatchFeatureExtractor
+
+
+class PrestoFeatureExtractor(PatchFeatureExtractor):
```

Comment:
is it intentional that all implementations just do `pass` ? Or is this placeholder code for now

Reply:
I will remove this as this should go on a separate repository. I would leave the other example for the quantile extraction as it illustrates well how to use a Feature Extractor in GFMAP. I renamed it's folder to `examples/feature_extractors` though